src/common/config/defaultPreferences.js

```diff
@@ -6,6 +6,20 @@
  * Default user preferences. End-users can change these parameters by editing config.json
  * @param {number} version - Scheme version. (Not application version)
  */
+
+const getDefaultDownloadLocation = () => {
+    switch (process.platform) {
+    case 'darwin':
+        return `/Users/${process.env.USER || process.env.USERNAME}/Downloads`;
+    case 'win32':
+        return `C:\\Users\\${process.env.USER || process.env.USERNAME}\\Downloads`;
+    case 'linux':
+        return `/home/${process.env.USER || process.env.USERNAME}/Downloads`;
+    default:
+        return '';
```

Comment:
most other OSs use a similar structure as linux. defaulting to `''` might not be the best option.

Reply:
I would prefer the absolute default to be '' because then there's no value and therefore the app default won't be set. I think what I could do is add some of the other platform types as a case here, like BSD and those ones.

Reply:
Looks like all other platforms are Unix-based, so the `/home/user/Downloads` path makes sense as the default.